manual_nodeinterface.py

```diff
@@ -0,0 +1,31 @@
+import grpc
```

Comment:
If this is about a test for the node interface, I would prefer an automaticunit test. Still, I think it should go in some directory